src/pages/components/file-uploader/accessibility.mdx

```diff
@@ -19,82 +19,51 @@ import {
-[W3C Web Content Accessibility Guidelines (WCAG) 2.1 Level A and AA success criteria](https://www.w3.org/TR/WCAG21/)
-and violations have been identified as high priority issues. This document will
-be updated when these accessibility issues are resolved.
+No accessibility annotations are needed for file uploaders, but keep these considerations in mind if you’re modifying Carbon or creating a custom component.
 
 </PageDescription>
 
 <AnchorLinks>
-  <AnchorLink>Accessibility considerations</AnchorLink>
-  <AnchorLink>Resources</AnchorLink>
-  <AnchorLink>Accessibility testing</AnchorLink>
+  <AnchorLink>What Carbon provides</AnchorLink>
+  <AnchorLink>Development considerations</AnchorLink>
 </AnchorLinks>
 
-## Accessibility considerations
-
-1. After the file uploader closes, focus should retain the user's point of
-   regard and return to the element that invoked it.
-2. The file uploader button can be activated using both the `Space` or `Enter`
-   key.
-3. The file uploader button should have text or label that accurately describes
-   the action of the button.
-
-## Resources
-
-- [IBM Accessibility Requirements](https://www.ibm.com/able/requirements/requirements/):
-  - [1.4.13 Content on Hover or Focus](https://www.ibm.com/able/requirements/requirements/#1_4_13)
-    (WCAG Success Criteria
-    [1.4.13](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html))
-  - [2.1.2 No Keyboard Traps](https://www.ibm.com/able/requirements/requirements/#2_1_2)
-    (WCAG Success Criteria
-    [2.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-trapping.html))
-  - [2.4.3 Focus Order](https://www.ibm.com/able/requirements/requirements/#2_4_3)
-    (WCAG Success Criteria
-    [2.4.3](https://www.w3.org/WAI/WCAG21/Understanding/focus-order))
-  - [3.2.2 On Input](https://www.ibm.com/able/requirements/requirements/#3_2_2)
-    (WCAG Success Criteria
-    [3.2.2](https://www.w3.org/TR/2016/NOTE-UNDERSTANDING-WCAG20-20161007/consistent-behavior-unpredictable-change.html))
-
-## Accessibility testing
-
-Automated, manual and screen reader accessibility verification test has been
-performed on the file uploader React Carbon Component.
-[WCAG 2.1 Level A and AA success criteria](https://www.w3.org/TR/WCAG21/) issues
-have been identified and the list of open accessibility violations is available
-in the Carbon Component GitHub repository.
-
-### Automated test
+## What Carbon provides
+
+Carbon bakes keyboard operation into its components, improving the experience of blind users and others who operate via the keyboard. Carbon incorporates many other accessibility considerations, some of which are described below.
+
+### Keyboard interactions
+
+Both variants of the file uploader provide buttons for uploading and removing files.
+The drop target “Drag and drop files here..." also provides conventional button
+keyboard interaction (`Tab` to reach; `Enter` or `Space` to activate). Once a file
```

Comment:
>Once a file has been added, it can be removed by activating the delete (“x”) button after each file name.

- In the Anatomy section on the Usage tab, we describe this as the Delete `x` icon. And on the Style tab we describe it as the Delete icon. The x on the usage tab is styled as code and we say icon instead of button. I think we should align on our terminology for it.

Reply:
I agree Carbon is all over the place in how this is discussed -- even within the Usage content it varies. My concern with calling this an 'icon' in the context of the keyboard accessibility is that it isn't operabler because it's an icon (which isn't necessarily a control) but because it's a button. My concern with capitalizing "delete" is that that is not the visible label of the button, which is what I think traditionally is capitalized. This is an odd button, but this helps from IBM Style:
> When you write about an item that is displayed in a graphical user interface, match the capitalization and spelling of the item in your writing. For example, if a label is displayed with initial uppercase letters or all uppercase letters, write the label the same way in the documentation

Reply:
Oh sorry, I wasn't concerned about capitalization, just saying icon versus button. If you think that using button for keyboard accessibility context makes more sense, than we can keep it as is.